### ISSUE TITLE
[BUG] Social Links aren't visible on main page

### ISSUE THREAD

**Issue by Thaodan** [Pelican-Elegant/elegant#493]
## Mandatory Step

- [ x] I am using latest production release of Elegant

## Recommended Step

- [x ] I can't my source code but I have linked my pelican configuration files to help developers reproduce the issue

## Description

**Describe the bug**
The social media links aren't visible on the main page.
**To Reproduce**
1. Open page
2. See

**Expected behavior**
The social media links are visible on the main page.

**Screenshots**
No but look at my page: https://thaodan.de

## Config
```
#!/usr/bin/env python
# -*- coding: utf-8 -*- #
from __future__ import unicode_literals


PATH = 'content'

THEME = 'themes/elegant'

TIMEZONE = 'Europe/Paris'

DEFAULT_LANG = 'de'

AG_SAVE_AS = ''
AUTHOR_SAVE_AS = ''
#DIRECT_TEMPLATES = ('index', 'categories', 'archives', 'search')
TIPUE_SEARCH_SAVE_AS = 'tipue_search.json'

RELATIVE_URLS = False

FEED_ALL_ATOM = 'feeds/all.atom.xml'
STATIC_PATHS = ['images', 'extra/favicon.ico']
EXTRA_PATH_METADATA = {
    'extra/favicon.ico': {'path': 'favicon.ico'}
}


# Feed generation is usually not desired when developing
FEED_ALL_ATOM = None
CATEGORY_FEED_ATOM = None
TRANSLATION_FEED_ATOM = None
AUTHOR_FEED_ATOM = None
AUTHOR_FEED_RSS = None
TYPOGRIFY = True
# Blogroll
#LINKS = (('Pelican', 'http://getpelican.com/'),
 #        ('Python.org', 'http://python.org/'),
  #       ('Jinja2', 'http://jinja.pocoo.org/'),
   #      ('You can modify those links in your config file', '#'),)

# Social widget
SOCIAL = (('Twitter', 'https://twitter.com/_Thaodan'),
          ('Github', 'https://github.com/Thaodan'),
          ('Gitlab', 'https://gitlab.com/Thaodan'))


TWITTER_USERNAME = '_Thaodan'
DEFAULT_PAGINATION = 10

# Uncomment following line if you want document-relative URLs when developing
#RELATIVE_URLS = True

PLUGIN_PATHS = ['/usr/lib/python3.7/site-packages/pelican/plugins']

PLUGINS = ['org_reader']


ORG_READER_EMACS_LOCATION = '/usr/bin/emacs'

# skip git directory
OUTPUT_RETENTION = [".git"]
DELETE_OUTPUT_DIRECTORY = True
```

**Comment by AWegnerGitHub** [comment]
These links show up on the article pages. I see it on your test page:

![icons](https://user-images.githubusercontent.com/6462439/63279086-fdfdc000-c26d-11e9-9233-6c35ef0f03af.png)

**Comment by Thaodan** [comment]
That no the issue. I'm asking about the first page at the route of the website.

**Comment by talha131** [comment]
@Thaodan this is by design. We don't show any sidebar item on the landing page. Instead, we have [a detailed "about me" section](https://elegant.oncrashreboot.com/write-about-me) that user can leverage to mention his social media profiles.

Like, "I am Talha Mansoor and I tweet at @talham_. You can view my Git projects at this link."

Let me know what you have in mind. If you want to show social links in the landing page, where would the fit?

**Comment by Thaodan** [comment]
OK that makes sense in a way. 
Me thought practical and wanted to resuse the social section.

Maybe show the social links like in the sidebar on the side or down below.

**Comment by talha131** [comment]
@Thaodan I looked into your request and tried to find out a which will make it easy to add social icons in the landing page without major changes. Unfortunately, I wasn't successful.

I investigated using [jinja2content](https://github.com/getpelican/pelican-plugins/tree/master/jinja2content). The pelican build started to fail as soon as I added it to the plugins. If it worked users would have been able to use `{% include '~/Repos/elegant/templates/_includes/social_links.html' %}` in their Markdown files where they wanted to place social icons.

Another possible way is to create a plugin and move `social_links.html` template to it. Then where ever it would see certain keyword like `$SOCIAL$`, it would replace it with the template. Currently, I don't have the resources to create and test such a plugin.

The last option is to optionally add social icons in the landing page template. There are a couple of ways how a landing page may look depending on the user settings.

1. List of recent articles is visible only
1. List of recent articles with My projects
1. "About me" with Recent articles
1. "About me" with Recent articles and My Projects

Social-links widget has to fit all the above use cases.

I invite you to open a PR for it if you think links to social profiles in "About me" is not enough.
